### PR TITLE
adapt UT to be cross architecture

### PR DESCRIPTION
**Pull request by oriyarde**
This change is [Reviewable](https://reviewable.io/reviews/ibm/ibm-block-csi-operator/60)

### CODE REVIEW

File: hack/update-crds.sh [hunk 1]
```diff
@@ -25,4 +25,4 @@ fi
 
 operator-sdk generate k8s
 
-operator-sdk generate openapi
+operator-sdk generate crds
```
Comment (27149chen) [inline hack/update-crds.sh]: please don't change this line.

File: build/ci/Dockerfile.unittest [hunk 2]
```diff
@@ -19,12 +19,14 @@ ENV GO111MODULE=on \
...
-	&& chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu \
+RUN RELEASE_VERSION=v0.16.0 \
+    && ARCH=$(uname -m) \
+    && SDK_ASSET=operator-sdk-${RELEASE_VERSION}-${ARCH}-linux-gnu \
+    && curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/${SDK_ASSET} \
+	&& chmod +x ${SDK_ASSET} \
 	&& mkdir -p /usr/local/bin/ \
-	&& cp operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu /usr/local/bin/operator-sdk \
-	&& rm operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+	&& cp ${SDK_ASSET} /usr/local/bin/operator-sdk \
```
Comment (igorgonibm) [inline build/ci/Dockerfile.unittest]: where SDK ASSET defined?

Comment (oriyarde) [reply build/ci/Dockerfile.unittest]: line 24

File: hack/update-crds.sh (hunk 1 above)
Comment (oriyarde) [reply hack/update-crds.sh]: OK, I opened a separate ticket CSI-1296

**Pull request merged**